packages/waku/src/router/client.ts

```diff
@@ -482,6 +491,18 @@ const Redirect = ({ to, reset }: { to: string; reset: () => void }) => {
       })
```

Comment:
Just curious: Is the setTimeout hack still needed with the new route change behavior?

Reply:
I just tried and there are e2e tests that break if I remove it.